AssertionError: (3 < 2) is wrong
